src/components/Notifications/NotificationFeed/NotificationFeed.js

```diff
@@ -42,6 +42,7 @@ class NotificationFeed extends React.Component {
     this.NOTIFICATION_FETCH_LIMIT = 10
     this.TOOLTIP_ID = 'react-autoql-notification-feed-tooltip'
     this.CHART_TOOLTIP_ID = 'react-autoql-notification-feed-chart-tooltip'
+    this.ALL_PROJECTS = 'All'
     // Open event source http connection here to receive SSE
```

Comment:
Can you remove these comments?

Reply:
I don't even know where does it come from, just removed it.